server/service/src/standard_reports.rs

```diff
@@ -25,7 +25,7 @@ pub struct StandardReports;
 impl StandardReports {
     // Load embedded reports
     pub fn load_reports(con: &StorageConnection) -> Result<(), anyhow::Error> {
-        for file in EmbeddedStandardReports::iter() {
+        for file in EmbeddedStandardReports::iter().filter(|name| name != ".DS_Store") {
```

Comment:
I think it's good to use: https://github.com/pyrossh/rust-embed?tab=readme-ov-file#include-exclude